jerry-core/lit/lit-strings.h

```diff
@@ -25,15 +25,81 @@
 #define LIT_BYTE_NULL (0)
 
 /**
+ * For the formal definition of Unicode transformation formats (UTF) see Section 3.9, Unicode Encoding Forms in The
+ * Unicode Standard (http://www.unicode.org/versions/Unicode7.0.0/ch03.pdf#G7404, tables 3-6, 3-7).
```

Comment:
Why do you use Unicode 7.0? Since @ruben-ayrapetyan used data from Unicode 3.0 tables, should we regenerate them for 7.0?